src/Microsoft.DotNet.ProjectJsonMigration/MSBuildExtensions.cs

```diff
@@ -27,14 +27,18 @@ public static bool IsEquivalentTo(this ProjectItemElement item, ProjectItemEleme
             // Different includes
             if (item.IntersectIncludes(otherItem).Count() != item.Includes().Count())
             {
+#if !DISABLE_TRACE
```

Comment:
Why introduce this? I think we could use the same verbose mechanism that we use for Reporter.Verbose.WriteLine. Check it out.